features/support/cucumber_rails_helper.rb

```diff
@@ -16,6 +16,7 @@ def install_cucumber_rails(*options)
     add_gem 'database_cleaner-active_record', '>= 2.0.0.beta2', group: :test if options.include?(:database_cleaner_active_record)
     add_gem 'factory_bot', '>= 3.2', group: :test unless options.include?(:no_factory_bot)
 
+    run_command_and_stop "bundle config --local path #{ENV['GITHUB_WORKSPACE']}/vendor/bundle" if ENV.key? 'GITHUB_WORKSPACE'
```

Comment:
A small optimisation to avoid installing gems on each Cucumber scenario: install the nested Rails project gems in a location that can be reused through out the build step.

Reply:
No idea if this is legit. If it is great work and find!

Reply:
@olleolleolle or @mvz could you dive into this with @orien whilst I'm busy / off on holiday. I'll un-assign my "needs changes" and let one of you approve / merge.

NB: Can we not merge this in for the next 2 days whilst I focus on cutting a 2.3.0 release.

Reply:
This optimisation is not super impressive as far as time saved in the build. I think it's better to avoid any confusion in this PR and just focus on the core content. I'll remove this change and raise it in a subsequent PR so it can be considered in isolation.